Include/internal/pycore_slp_pystate.h

```diff
@@ -70,10 +70,22 @@ typedef struct {
  */
 struct _stackless_runtime_state {
     /*
-     * flag whether the next call should try to be stackless.
-     * The protocol is: This flag may be only set if the called
+     * try_stackless: flag whether the next call should try to be stackless.
+     *
+     * Possible values:
+     *   0: don't be stackless
+     *   1: any called C-function shall try to be stackless.
+     *   other: only the C-function with address try_stackless shall try to
+     *          by stackless.
+     *
```

Comment:
This is the salient change, right? the *other* value?  Maybe it should be documented here why
 - it is necessary (something about vectorcall functions being properties)
 - how it works, in particular, how we can do it without knowing if the called function supports the protocol or not.